nixos/modules/services/misc/etesync-dav.nix

```diff
@@ -0,0 +1,69 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.etesync-dav;
+in
+  {
+    options.services.etesync-dav = {
+      enable = mkEnableOption "etesync-dav";
+
+      package = mkOption {
```

Comment:
It looks like we only have 1 variant of this package. I think this option might not provide that much value.

Reply:
I've been adding it to all modules I contribute because I figured it was a best practice. But good point, I will remove it.